### PR TITLE
ci: experiments

### PR DESCRIPTION
**Pull request by henryiii**
This might have an issue, noticed when working on the action.﻿

### CODE REVIEW

**Comment by henryiii** [comment]
```
Run joerick/pr-labels-action@v1.0.6
No labels found
```

?

**Comment by henryiii** [comment]
Curious, this works, will continue to investigate why my run failed on 20.04. Actually, I am using a custom manylinux1 image that might have issues with 20.04 somehow?

I dropped the custom action and am just using native syntax for accessing labels, and fixed the label name. Since 20.04 is about to become the default, maybe merging this PR is a good idea? Should we update the examples?

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -43,12 +43,8 @@ jobs:
       run: |
         choco install vcpython27 -f -y
 
-    - name: Get PR labels
-      id: pr-labels
-      uses: joerick/pr-labels-action@v1.0.6
```
Comment (YannickJadoul) [inline .github/workflows/test.yml:48]: @joerick, this seems like an interesting/important consequence towards your custom action/repository?

**Comment by YannickJadoul** [comment]
Shouldn't/couldn't we just grow along with `-latest`? Of course, Ubuntu 22.04 is still quite far away, so it won't really make that much of a difference, afterwards.

**Comment by henryiii** [comment]
Since `-latest` is about to change, I at least want(ed) to test with it.

File: .github/workflows/test.yml (hunk 1 above)
Comment (henryiii) [reply .github/workflows/test.yml:48]: Maybe this pre-dates the syntax for accessing labels?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -43,12 +43,8 @@ jobs:
...
-
     - name: Sample build
-      if: contains(steps.pr-labels.outputs.labels, ' ci-sample-build ')
+      if: "contains(github.event.pull_request.labels.*.name, 'CI: Sample build')"
```
Comment (henryiii) [inline .github/workflows/test.yml:47]: We should also decide what we want to do for non pull_request builds. Should we only build samples if requested on pull_requests? It should also be possible via `workflow_dispatch`, what about on master?

Comment (YannickJadoul) [reply .github/workflows/test.yml:47]: What if we set up some kind of weekly cron trigger? I had already forgotten there was this label to trigger things :-/ Once a week (or so) doesn't seem to crazy when it comes to using resources?

**Comment by henryiii** [comment]
Trying to see if I can get `cibuildwheel` to run directly in powershell. Nope, that didn't trigger what I've been seeing.

File: .github/workflows/test.yml (hunk 1 above)
Comment (joerick) [reply .github/workflows/test.yml:48]: No idea why it wasn't working, it [still works on my repo](https://github.com/joerick/pr-labels-action/pull/5/checks?check_run_id=1717049797). Labels can be missed depending on when they're added - they're snapshotted when the build was triggered - perhaps the label was added after the build was triggered?

Comment (henryiii) [reply .github/workflows/test.yml:48]: I restarted the build after adding the label, it should have been picked up? Though I didn't try closing and reopening.

Is there a benefit to the action over just using an if?

**Comment by henryiii** [comment]
Closing so as to not take focus off of more important PRs. Though still wonder if this can be simplified with a native if instead of a custom action.

**Pull request closed without merging**